cpp/common/env.h

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) ZeroC, Inc.
+
+#include <optional>
+#include <string>
+
+namespace common
```

Comment:
I would call it `Env`, not common. With a PascalName namespace since it's our convention.

Same for the file => Env.h.

Reply:
Fixed.